uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/performance/ProcessInfo.java

```diff
@@ -0,0 +1,49 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.performance;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+/**
+ * An object that stores the current process information
+ */
+@Data
+@Accessors(prefix = "m")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ProcessInfo {
+    private int mPid;
+    private long mTotalSystemMemory;
+    private long mUsedSystemMemory;
+    private long mFreeSystemMemory;
+    private long mAvailableSystemMemory;
+    private int mUid;
+    private double mCpuUsage;
+    private double mMemoryUsage;
```

Comment:
nit: final to all these fields

Reply:
Choose to keep only the `@NoArgsConstructor`.  So we can't set `final` here.